src/Systmm.f90

```diff
@@ -357,7 +361,7 @@ SUBROUTINE SYSTMM(temp_file)
                   , nresx, dt_comp)
 
                 call energy(T_epil(nresx), q_surf, res_end_node(nresx))
-
+[ENERGY SHOULD BE CALLED IN THE MODULE THAT DOES RIVER SEGMENTS AND IN THE MODULE THAT DOES RESERVOIRS - JRY]
```

Comment:
It is called twice.  In the previous res_to_rbm_modular version of code, energy was called in line 287 (for river) and line 376 (for reservoir).